count.py

```diff
@@ -0,0 +1,35 @@
+from datetime import timedelta, date
+
+# Checking last days count for inventory
+def countCheck(source, currSource):
+    count = 0
+    today = date.today()
+    today = today - timedelta(days=1)
+    yesterday = today - timedelta(days=1)
+    dayBefore = today - timedelta(days=2)
+    d2 = yesterday.strftime("%Y-%m-%d")
+    d3 = dayBefore.strftime("%Y-%m-%d")
```

Comment:
Avoid naming variables like d2 and d3

Reply:
changed names like d2 and d3 to proper name